jobsdb/jobsdb.go

```diff
@@ -2390,26 +2409,26 @@ func (jd *HandleT) doStoreJobsInTx(ctx context.Context, tx *sql.Tx, ds dataSetT,
 
 func (jd *HandleT) storeJob(ctx context.Context, tx *sql.Tx, ds dataSetT, job *JobT) (err error) {
 	sqlStatement := fmt.Sprintf(`INSERT INTO %q (uuid, user_id, custom_val, parameters, event_payload, workspace_id)
-	                                   VALUES ($1, $2, $3, $4, $5, $6) RETURNING job_id`, ds.JobTable)
+		VALUES ($1, $2, $3, $4, $5, $6) RETURNING job_id`, ds.JobTable)
 	stmt, err := tx.PrepareContext(ctx, sqlStatement)
-	jd.assertError(err)
-	defer func() { _ = stmt.Close() }()
+	if err != nil {
+		return err
+	}
+	defer stmt.Close()
```

Comment:
I have seen other people usually follow the convention of using 
defer func() { _ = stmt.Close() }()
I have been told this makes it clear that we are ignoring the return value and is the preferable style. We should probably stick to that convention. You might want to check with @fracasula  (who might be able to advise better)

Reply:
done.